galleries/plot_types/3D/trisurf3d_simple.py

```diff
@@ -1,7 +1,7 @@
 """
-======================
-Triangular 3D surfaces
-======================
+=====================
+plot_trisurf(x, y, z)
+=====================       
```

Comment:
This linting failure must be fixed.
```suggestion
=====================
```